### PR TITLE
Adding cost validation section

### PR DESCRIPTION
**Pull request by goeln**
### What does this PR do?

Adds necessary set up permissions

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/cloud_cost_management/_index.md [hunk 1]
```diff
@@ -120,6 +120,12 @@ Attach the new S3 policy to the Datadog integration role.
 5. Click **Attach policy**.
 
 **Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
+
+### Provide linked account access for cost validation
+
+The Datadog Cloud Cost Management product matches cost calculations against AWS' Cost Explorer to ensure complete cost accuracy. If your Cost and Usage Report (CUR) provided to Datadog is from a Linked Account, please allow Linked Account Access, Linked Account Refunds and Credits, and Linked Account Discounts under AWS Cost Management preferences.
```
Comment (cnharris10) [inline content/en/cloud_cost_management/_index.md:126]: Would highly suggest editing to something like ~"matches cost calculations _periodically_".  Customers pay for us requesting Cost Explorer, so don't want them to think that we are overloading more cost on them.

Comment (kayayarai) [inline content/en/cloud_cost_management/_index.md]:
```suggestion
Cloud Cost Management matches cost calculations against the AWS Cost Explorer to ensure complete cost accuracy. If your CUR is from a Linked Account, in the AWS Cost Management preferences, enable **Linked Account Access**, **Linked Account Refunds and Credits**, and **Linked Account Discounts**.
```

Comment (goeln) [reply content/en/cloud_cost_management/_index.md]: done

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/cloud_cost_management/_index.md [hunk 2]
```diff
@@ -120,6 +120,12 @@ Attach the new S3 policy to the Datadog integration role.
 5. Click **Attach policy**.
 
 **Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
+
+### Provide linked account access for cost validation
+
+The Datadog Cloud Cost Management product periodically matches cost calculations against AWS' Cost Explorer to ensure complete cost accuracy. If your Cost and Usage Report (CUR) provided to Datadog is from a Linked Account, allow Linked Account Access, Linked Account Refunds and Credits, and Linked Account Discounts under AWS Cost Management preferences.
```
Comment (kayayarai) [inline content/en/cloud_cost_management/_index.md:126]:
```suggestion
Cloud Cost Management product periodically matches cost calculations against the AWS Cost Explorer to ensure complete cost accuracy. If your CUR is from a Linked Account, in the AWS Cost Management preferences, enable **Linked Account Access**, **Linked Account Refunds and Credits**, and **Linked Account Discounts**.
```

File: content/en/cloud_cost_management/_index.md [hunk 3]
```diff
@@ -120,6 +120,12 @@ Attach the new S3 policy to the Datadog integration role.
 5. Click **Attach policy**.
 
 **Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
+
+### Provide linked account access for cost validation
+
+Cloud Cost Management product periodically matches cost calculations against the AWS Cost Explorer to ensure complete cost accuracy. If your CUR is from a Linked Account, in the AWS Cost Management preferences, enable **Linked Account Access**, **Linked Account Refunds and Credits**, and **Linked Account Discounts**.
```
Comment (kayayarai) [inline content/en/cloud_cost_management/_index.md:126]:
```suggestion
Cloud Cost Management periodically matches cost calculations against the AWS Cost Explorer to ensure complete cost accuracy. If your CUR is from a Linked Account, in the AWS Cost Management preferences, enable **Linked Account Access**, **Linked Account Refunds and Credits**, and **Linked Account Discounts**.
```

**Pull request closed without merging**